README.md

```diff
@@ -49,6 +49,10 @@ The port on which Solr will run.
 
 Memory settings for the JVM. These should be set as high as you can allow for best performance and to reduce the chance of Solr restarting itself due to OOM situations.
 
+    solr_timezone: "UTC"
+
+Solr used timezone
```

Comment:
This is a very detailled article about it and problems you can face if not using java to feed solr : 
https://bitbach.wordpress.com/2015/06/22/solr-2-oracle-date-indexing-timezone-handling-and-probable-issues/#more-1427

at the end i guess changing the SOLR_TIMEZONE does change the default TZ interpretation for non precised TZ datetimes (ex: php or string datetimes). I have to run some tests to confirm it.